autodesk_dwgtrueview/test.sikuli/test.py

```diff
@@ -0,0 +1,55 @@
+script_path = os.path.dirname(os.path.abspath(sys.argv[0])) 
+include_path = os.path.join(script_path, os.pardir, os.pardir, "!include", "util.sikuli")
+sys.path.append(include_path)
+import util
+reload(util)
+addImagePath(include_path)
+
+dwgfile_path = os.path.join(script_path, os.pardir, "resources", "blocks_and_tables_-_metric.dwg")
+
+setAutoWaitTimeout(30)
+util.pre_test()
+
+# Test of `turbo run`.
+wait("app-loaded.png")
```

Comment:
I tried several times, and the test always failed at this line. The app did launch. So maybe the waiting time needs to be increased.